src/api/routes/users.ts

```diff
@@ -55,14 +55,41 @@ router.patch("/:userId", (req: Request, res: Response, next: NextFunction) => {
                 last_name = user.last_name,
                 email = user.email,
             } = jsonData;
-
-            res.status(200).json({ first_name: first_name, last_name: last_name, email: email });
+        res.status(200).json({ first_name: first_name, last_name: last_name, email: email });
         })
         .catch((err: string) => {
             res.status(404).json({ message: "user not found" });
         });
 });
 
+router.post('/', (req: Request, res: Response, next: NextFunction) => {
+    var userData=req.body
+    //validating parameters
+    var error="required ";
+    if (!userData.first_name) {
+        error +=" first_name "
+    }if(!userData.last_name){
+        error +=" last_name "
+    }if(!userData.email){
+        error +=" email "
+    }
```

Comment:
Please change to a formatted string instead of string concatenation.
Example:
```js
if (!userData.first_name) {
        required_field += "first_name"
...    
...
res.status(400).json({error: "required ${required_field}"})
```